pgx/src/pgx.ml

```diff
@@ -1325,67 +1327,111 @@ module Make (Thread : IO) = struct
 
   let execute ?(params=[]) db query =
     Prepared.(with_prepare db ~query ~f:(fun s ->
-      execute s ~params))
-
-  let execute_iter ?(params=[]) db query ~f =
-    Prepared.(with_prepare db ~query ~f:(fun s ->
+        execute s ~params))
+
+  open CamlinternalFormatBasics
```

Comment:
Here's where the interesting part starts. I'll redo the diff to get these whitespace changes out soon.